custom/icds_reports/tests/reports/test_awc_reports.py

```diff
@@ -2185,15 +2153,15 @@ def test_awc_report_beneficiary_82f33fa1_2aec_45ba_8d6c_d3ca9f50ab73(self):
                     "recorded_height": 0,
                     "fully_immunized": "No",
                     "person_name": "Name 4445",
-                    "sex": "F",
```

Comment:
Why was this removed? Were these columns unused?

Reply:
@emord This columns are no longer used in the beneficiary list report